### ISSUE TITLE
Is NVDA compatible with no code, low code tools?

### ISSUE THREAD

**Issue by NVCDevelopmentTeam** [ElijT/Awesome-no-code-tools#3]
I'm using NVDA with no code, low code tools and I've found it's not accessible to the visually impaired at all. What do you think about this issue, you can click on this link so we can discuss together. I made the discussion in this link:
https://github.com/nvaccess/nvda/discussions/12479#discussion-3387990
I hope things work out for the time being.

**Comment by NVCDevelopmentTeam** [comment]
I think this is the biggest difficulty that I and every disability are facing. So I want my discussion to be public so everyone on github can participate. From there, it is possible to come up with a more reasonable way to solve this problem. Help people understand the problem that needs to be handled.
